src/Stryker.Core/Stryker.Core/Mutators/LinqMutator.cs

```diff
@@ -40,30 +40,54 @@ public LinqMutator()
                 { LinqExpression.ThenBy, LinqExpression.ThenByDescending },
                 { LinqExpression.ThenByDescending, LinqExpression.ThenBy }
             };
+            RequireFilter = new Dictionary<LinqExpression, bool>
```

Comment:
The name RequireFilter doesn't cover what this does. Maybe:
```
RequireArgumentsFilter = new Dictionary<LinqExpression, bool>
```

Reply:
This could also be an array instead of dictionary. Just add only the Expression types that have a "true" value and check if it is present in the array. That would shorten this collection a lot!

Reply:
done